core/geometry/src/serialization/BGFBAccessors.ts

```diff
@@ -4948,11 +5150,20 @@ expectedClosure(): number {
   return offset ? this.bb!.readInt32(this.bb_pos + offset) : 0;
 }
 
+/**
+ * @param TaggedNumericData= obj
+ * @returns TaggedNumericData|null
+ */
+taggedNumericData(obj?: TaggedNumericData): TaggedNumericData|null {
+  const offset = this.bb!.__offset(this.bb_pos, 40);
+  return offset ? (obj || new TaggedNumericData()).__init(this.bb!.__indirect(this.bb_pos + offset), this.bb!) : null;
+}
+
 /**
  * @param flatbuffers.Builder builder
  */
 static startPolyface(builder: flatbuffers.Builder) {
-  builder.startObject(18);

```

Comment:
What's this hard-code 18/19 about?

Reply:
This is machine generated flatbuffer access codes with offsets.